src/foam/nanos/export/GoogleSheetsOutputter.js

```diff
@@ -89,8 +89,14 @@ foam.CLASS({
               propValues.push(obj[columnMethadata[i].columnName].toString().substring(0, 8));
               columnMethadata[i].perValuePatternSpecificValues.push(obj[columnMethadata[i].columnName].toString().substring(8));
             }
-            else
-              propValues.push(obj[columnMethadata[i].columnName].toString());
+            else {
+              if( typeof obj[columnMethadata[i].columnName] === "function" )
```

Comment:
space after 'if'.
If this is the case, won't it always be the case and then couldn't we just not include this column?

Reply:
Fixed